src/modules/github/github.actions.js

```diff
@@ -0,0 +1,45 @@
+import GitHubAPI from "./github.api";
+import types from './github.types';
+
+const api = new GitHubAPI();
+
+const getRepos = repos => {
```

Comment:
Nie jest to bardziej `setRepos`? Jest parametr, przez który przekazujesz repozytoria do ustawienia.